packages/calcite-components/src/components/menu-item/menu-item.scss

```diff
@@ -3,8 +3,12 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
+ * @prop --calcite-menu-item-action-background-color: defines the background color of an action sub-component inside a menu-item.
+ * @prop --calcite-menu-item-action-indicator-color: defines the indicator color of an action sub-component inside a menu-item.
  * @prop --calcite-menu-item-action-menu-border-color: The border color of the sub-component.
  * @prop --calcite-menu-item-action-menu-text-color: The text color of the sub-component.
+ * @prop --calcite-menu-item-action-text-color: defines the text color of an action sub-component inside a menu-item.
```

Comment:
A user can't enable text on an Action rendered within Menu, I think we can remove this. I think we'd just need to expose the icon color here.

We might want to add `text` and `icon` to Menu Item itself, but outside the scope here.